Fix up to work with TF v1.0 API.

darrengarvey opened the pull request:
rnn_cell has moved to tensorflow.contrib.rnn. This change was made in
v0.12 [1].

[1] See https://github.com/tensorflow/tensorflow/commit/37fbebdd6c3c8f274896cc36e6feb5b7e2097a59

cghawthorne commented on magenta/models/rl_tuner/rl_tuner_ops.py:
```diff
@@ -21,6 +21,7 @@
 
 import numpy as np
 import tensorflow as tf
+from tensorflow.contrib.rnn import rnn_cell
```
to be consistent with the rest of our code, we'd prefer to just have one `import tensorflow as tf`, and then have the below references be things like `tf.contrib.rnn.LSTMCell`.

darrengarvey replied:
I should have noticed that was your preferred style. I've made the change: now, do you prefer force-pushing the fixup to keep the PR clean or add an additional commit on top?

darrengarvey replied:
I've pushed as an extra commit, but can squash down if that's how you do it.

cghawthorne replied:
Extra commits are fine. We'll squash before we merge.

cghawthorne commented on magenta/models/rl_tuner/rl_tuner_ops.py:
```diff
@@ -297,15 +297,17 @@ def make_rnn_cell(rnn_layer_sizes, state_is_tuple=False):
   cells = []
   for num_units in rnn_layer_sizes:
-    cell = tf.nn.rnn_cell.LSTMCell(
+    cell = tf.contrib.rnn.rnn_cell.LSTMCell(
```
Need to remove the extra rnn_cell so it's just tf.contrib.rnn.LSTMCell

darrengarvey commented:
Removed the superfluous references to `rnn_cell`. I hadn't noticed that ugliness became unnecessary.

I also fixed up a doc reference to `RnnCell`, which AFAICT should have been `RNNCell`.
